test/e2e/nodepool_test.go

```diff
@@ -119,11 +119,19 @@ func TestNodePool(t *testing.T) {
 					},
 					{
 						name: "TestNodePoolPrevReleaseN1",
-						test: NewNodePoolPrevReleaseCreateTest(hostedCluster, globalOpts.N1MinorReleaseImage, clusterOpts),
+						test: NewNodePoolPrevReleaseCreateTest(hostedCluster, globalOpts.N1MinorReleaseImage, clusterOpts, true),
 					},
 					{
 						name: "TestNodePoolPrevReleaseN2",
-						test: NewNodePoolPrevReleaseCreateTest(hostedCluster, globalOpts.N2MinorReleaseImage, clusterOpts),
+						test: NewNodePoolPrevReleaseCreateTest(hostedCluster, globalOpts.N2MinorReleaseImage, clusterOpts, true),
+					},
+					{
+						name: "TestNodePoolPrevReleaseN3",
+						test: NewNodePoolPrevReleaseCreateTest(hostedCluster, globalOpts.N3MinorReleaseImage, clusterOpts, true),
+					},
+					{
+						name: "TestNodePoolPrevReleaseN4",
+						test: NewNodePoolPrevReleaseCreateTest(hostedCluster, globalOpts.N4MinorReleaseImage, clusterOpts, false),
```

Comment:
Would we always expect this test to pass? If the release is outside of the supported version skew, it would be possible that nodes do not become available, and that would be ok, no?